NguyenThuPhuong/phuong-app/resources/views/my-directory/product-single-ms.blade.php

```diff
@@ -155,4 +159,32 @@
             </div>
         </div>
     </section>
+
+@section('script')
+  <script type="text/javascript">
+    $('.add-to-cart').click(function(e){
+        e.preventDefault();
+        var currentQuantity = parseInt( $('#numberItem').text());
+        var addQuantity = parseInt($('#quantity').val());
+        var newQuantity = currentQuantity + addQuantity;
+        $('#numberItem').text(newQuantity);
+
+        Swal.fire(
+            'Add to cart successfully !' 
+        )
+    });
+
+    $('.add-to-heart').click(function(e){
+        e.preventDefault();
+        var currentQuantity = parseInt( $('#numberLove').text());
+        var newQuantity = currentQuantity + 1;
+        $('#numberLove').text(newQuantity);
+
+        Swal.fire(
+            'Add to favorites successfully !' 
+        )
+    });
+  </script>
+@endsection
```

Comment:
phần code này giống hệt đoạn bên trên, mình hoàn toàn có thể đưa vào 1 file, sau đó include vào thì đỡ phải viết 2 lần